credential-registry/client/tob-web/src/app/data-types.ts

```diff
@@ -215,50 +211,50 @@ export namespace Model {
     }
 
     get link() {
-      if(this.topic) {
-        return this.topic.link.concat(['/cred/', ''+this.id]);
+      if (this.topic) {
+        return this.topic.link.concat(["/cred/", "" + this.id]);
       }
     }
   }
 
   export class CredentialFormatted extends Credential {
-    static extPath = 'formatted';
+    static extPath = "formatted";
   }
 
   export class CredentialSearchResult extends Credential {
-    static resourceName = 'search/credential/topic';
+    static resourceName = "search/credential/topic";
   }
 
   export class CredentialFacetSearchResult extends Credential {
-    static resourceName = 'search/credential/topic/facets';
+    static resourceName = "search/credential/topic/facets";
   }
 
   export class CredentialVerifyResult extends BaseModel {
     success: boolean;
     result: any;
 
-    static resourceName = 'credential';
-    static extPath = 'verify';
+    static resourceName = "credential";
+    static extPath = "verify";
 
     get claims() {
       let ret = [];
-      if(typeof this.result === 'object' && this.result.presentation) {
+      if (typeof this.result === "object" && this.result.presentation) {
         let attrs = this.result.presentation.requested_proof.revealed_attrs;
-        for(let k in attrs) {
-          ret.push({name: k, value: attrs[k].raw});
+        for (let k in attrs) {
+          const attributeName = this.result.presentation_request.requested_attributes[k].name;
+          ret.push({ name: attributeName, value: attrs[k].raw });
         }
```

Comment:
This is the only change in this file, the rest is just the code being formatted.
Do we have standard formatting settings for our JS/TS code?